Shivani_Bansal/calculator/Calculator.css

```diff
@@ -0,0 +1,82 @@
+*,
+*:after,
+*:before{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    
+}
+
+/* .div1{ */
+    /* display: flex; */
+    /* align-items: center;
+    justify-content: center; */
+    /* margin: auto;  */
+    /* padding: 0px 34rem; */
+    
+    
+
+/* } */
+
+
+/* .inputArea{
+width: 5rem;
+} */
+
+/* .keypad{
+    display: flex;
+    width: 5rem;
+}  */
+
+.main{
+    width: 50%;
+    margin: auto;
+    padding: 1rem;
+    border: 1px solid black;
+    box-shadow: 6px 7px 5px 10px black;
+    min-width: 500px;
+}
+
+button{
+    width: 100%;
+    padding: 1rem;
+    font-size: 2rem!important;
```

Comment:
get rid of Important rule

Reply:
I've used !important to overwrite foreign CSS i.e bootstrap here.